frontend/src/features/documents/list/DocumentListContainer.tsx

```diff
@@ -0,0 +1,41 @@
+import { DocumentRelationshipType } from 'constants/documentRelationshipType';
+import { Api_DocumentRelationship } from 'models/api/Document';
+import { useEffect, useState } from 'react';
+
+import { useDocumentRelationshipProvider } from '../hooks/useDocumentRelationshipProvider';
+import DocumentListView from './DocumentListView';
+
+interface IDocumentListContainerProps {
+  parentId: number;
+  relationshipType: DocumentRelationshipType;
+}
+
+const DocumentListContainer: React.FunctionComponent<IDocumentListContainerProps> = props => {
```

Comment:
this is the component that was missing that retrieves the documents from the backend. All other changes fill the gap for this to work